backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -590,8 +590,13 @@ public String activateOrganizationWithSingleUser(Organization org) {
 
   @Override
   public List<String> fetchAdminUserEmail(Organization org) {
-    var admins = getOrgAdminUsers(org);
-    return admins.stream().map(u -> u.getProfile().getLogin()).toList();
+    try {
+      List<User> admins = getOrgAdminUsers(org);
+      return admins.stream().map(u -> u.getProfile().getLogin()).toList();
+    } catch (IllegalGraphqlArgumentException e) {
+      log.error("error fetching admin by email from Okta API");
+      return List.of();
```

Comment:
love what we did here! yay for pushing errors down closer to the offending code :)